A3.md

```diff
@@ -111,14 +112,16 @@ grpc_json* grpc_channel_tracer_get_trace(grpc_channel_tracer* tracer);
 
 ### Format of Exported Data
```

Comment:
This subsection should probably be moved up into the "Proposal" section above, since it's not implementation-specific -- we should use the same data format in all implementations.